src/devices/OpenHardwareMonitor/OpenHardwareMonitor.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This may raise a question as so far netcore2.1 is used. I'm personally very happy to see netstandard2.0 btw

Reply:
I think most projects where changed some time ago, and apparently I just copied this over from an older project state. But since it's only the test package, it doesn't really matter (these dll's are not shipped).